elixir/apps/domain/lib/domain/auth/subject.ex

```diff
@@ -7,7 +7,7 @@ defmodule Domain.Auth.Subject do
   @type permission :: Permission.t()
 
   @type t :: %__MODULE__{
-          identity: identity(),
+          identity: identity() | nil,
```

Comment:
Just making sure I follow -- this means a service account is not tied to any particular user correct?